packages/manager/modules/pci/src/projects/project/data-processing/job-status/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+
+import dataProcessingJobStatusComponent from './job-status.component';
+
+const moduleName = 'OvhManagerDataProcessingJobStatusComponent';
```

Comment:
Is this module really needed?

It's only used once and it's just here to display a `<span class="oui-status">`.

Reply:
Component is used in main job list and in detailed dashboard, removing it would imply code duplication.